reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -139,14 +143,14 @@ abstract class AbstractNoticeReporter : Reporter() {
         return result.toSortedMap()
     }
 
-    private fun writeNoticeReport(noticeReport: NoticeReport, outputFile: File) {
+    private fun writeNoticeReport(noticeReport: NoticeReport, copyrightGarbage: CopyrightGarbage, outputFile: File) {
```

Comment:
Before this function had a single responsibility of writing the report model to a file.
Now it also does a transformation which makes it harder to test.
How about passing the cleaned up finding in here instead of the garbage?

Reply:
my main point was responsibilty of that function, anyhow - it seems again a not too important detail, let's leave this for now.